trade-finance-manager-ui/server/helpers/destroy-session-and-redirect-to-start.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Destroys a user's session and redirects them to the start page.
+ * @param {Object} req
+ * @param {Object} res
+ * @returns {void}
+ */
+function destroySessionAndRedirectToStart(req, res) {
+  console.info('Invalid user JWT destroying session');
+  req.session.destroy(() => {
+    res.redirect('/');
+  });
+}
+
```

Comment:
Unit test

Reply:
FYI this is from `portal` microservice, maybe it has Unit tests already.

Reply:
Added Unit test, I think it is not from `portal`.